envi/codeflow.py

```diff
@@ -287,8 +286,6 @@ def addEntryPoint(self, va, arch=envi.ARCH_DEFAULT):
         self._funcs[va] = True
         calls_from = self.addCodeFlow(va, arch=arch)
         self._fcalls[va] = calls_from
-        # TODO: this ones spams. A lot.
-        # logger.debug('addEntryPoint(0x%x): calls_from: %r', va, calls_from)
```

Comment:
This has been commented out for quite a while. I'd say we should probably discuss it further if we want it back in (it lives in git after all, it's not hard to ressurrect). 

But I've been rethinking things and I'm still generally unhappy with MIRE and it's existence and the lack of consistency of messages and the general pain of use.  Throwing more messages back in the queue doesn't help that.

Reply:
how about this:  please leave it in there because i'm often uncommenting it for troubleshooting architecture things.
it's been commented out but used.